drivers/at24cxxx/at24cxxx.c

```diff
@@ -69,7 +69,7 @@
 #endif
 
 static
-int _read(const at24cxxx_t *dev, uint32_t pos, void *data, size_t len)
+int _read_page(const at24cxxx_t *dev, uint32_t pos, void *data, size_t len)
```

Comment:
I am a bit against naming this `_read_page()` because this is not what it does. It just reads.
So I would keep the name and wrap it in some `_read_max()` and substitute calls to `read()` with a loop of `_read_max()`.

Reply:
I agree. This was due to the lack of a better name and just matching the corresponding function for MTD. The issue is that `at24cxxx_read` only returns `AT24CXXX_OK` or error, but we need the amount of bytes read. For two proposals see the updated code.